File: All Topics/3101-3150/3110-ScoreofaString.ts
const s = "hello"

/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function (s: string) {
  const sLength = s.length
  let sum = 0

  for (let index = 0; index < sLength - 1; index++) {
    sum += Math.abs(s.charCodeAt(index) - s.charCodeAt(index + 1))
  }
  return sum
}

console.log(scoreOfString(s))

File: All Topics/3101-3150/3110-ScoreofaString.java
class ScoreofaString {
    public int scoreOfString(String s) {
        
int result=0;
for (int i = 1; i < s.length(); i++) {
result+=Math.abs(Character.codePointAt(s, i)-Character.codePointAt(s, i-1));
}

return result;
    }
}